package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "marvin",
+  "version": "0.0.0",
+  "description": "An API driven skin for MediaWiki",
+  "scripts": {
+    "start": "run-p -n dev:*",
+    "dev:client": "NODE_ENV=development webpack -w",
+    "dev:server": "NODE_ENV=development nodemon -w dist/client -w src/server/ src/server/index.js",
+    "prod:build": "NODE_ENV=production webpack -p",
+    "format": "prettier --write 'src/**/*.js'",
+    "lint": "eslint 'src/**/*.js'",
+    "test": "npm run lint",
+    "precommit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.js": [
+      "npm run test"
+    ]
+  },
+  "repository": {
```

Comment:
Cool! 👌 I didn't know about that! I just read https://docs.npmjs.com/files/package.json#repository and it seems it is only for the shortcuts. Given we are going to have to move repository to gerrit (☹️), I say we don't touch it for now, and just change the URL when we move away